x-pack/plugins/maps/public/classes/styles/ems/components/ems_vector_tile_style_editor.tsx

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { EuiFormRow, EuiColorPicker } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+interface Props {
+  color: string;
+  onEMSVectorTileColorChange: ({ color }: { color: string }) => void;
```

Comment:
How about renaming to just onColorChange? onEMSVectorTileColorChange is a little verbose.